presto-docs/src/main/sphinx/release/release-0.257.rst

```diff
@@ -0,0 +1,29 @@
+=============
+Release 0.257
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix scheduling issue in join spill to improve memory used by unspilled data.
```

Comment:
Improve memory usage of queries spilling in the join operator.

Reply:
done